process/TAC-Decision-Process.md

```diff
@@ -0,0 +1,42 @@
+
+ !! ADD link to TAC election process !!
+
+The TAC has a chairperson and vice-chairperson that are elected by the TAC members annually and are responsible for organizing and coordinating the TAC to conduct the technical management and oversight of the foundation's Technical Initiatives.  The TAC chair acts as a standing voting member of the Governance Committee (GC), and the vice-chair acting as their alternate/delegate.  A second TAC member, as determined by the TAC, is also included as a voting member of the GC. 
+
+Broadly, there are three main methods that the TAC will use to make decisions and ensure those decisions are appropriately documented:
+
+## 1.) Simple and/or non-consequential changes ##
+For asynchronous work items that do not have long-term/strategic consequences, the TAC will use lazy consensus to determine whether to adopt the proposal and ensure that it will be properly documented.  Issues and PRs should be created within the TAC GitHub repository and assigned to the TAC group for review.  Provided that at least two TAC members agree, support, and approve the suggested change, there are no objections raised by any TAC members, and no less than one week has transpired since the TAC was notified of the change, the proposal will be merged and adopted.
+
+This includes such things as documentation updates/minor changes
+
+
+## 2.) For work items discussed at the periodic TAC calls ##
+The TAC must have quorum (majority of members present) and a simple majority of the members present must approve of a proposal or topic.  This decision will be captured in the TAC meeting notes as well as an issue within the TAC repository. 
+
+
+
+## 3.) For  major items that require a formal vote ##
+An issue or PR must be created within the TAC repository.  TAC members will be asked via the TAC mailing list to review, comment, and decline/approve the item within the GitHub issue/PR.  The TAC will be given 10 business days to review and consider the request.  These topics ideally should also be presented at a TAC meeting and publicly discussed/debated prior to the closing of voting.  
+
+This includes items such as changes to the TAC technical vision, Foundation strategy, WG/project adoption, funding requests, adopting new Working Group, etc.).
+
+# Pull request types #
+
+Every PR must be categorized using one of the following <type> values. The purpose is twofold: to make it easier for readers to understand the scope of the PR at a glance, and to allow us to adjust the minimum review period and number of approvers based on how sensitive the PR is.
+
+Use the closest entry in the table that applies, selecting the first one if multiple apply. If you are not sure which type to use, take a guess and a maintainer will update if needed.
+
+|  Type  | Meaning | Waiting period |	# Approvers |
+| ------ | ------- | -------------- | ----------- |
+| TI lifecycle | A new TI or updated TI lifecycle stage. | 10d | 7 |
```

Comment:
I've been assuming this was 5 (a majority) not 7 (a supermajority) - I just want to clarify if you intended a majority or supermajority.

Reply:
Interesting.  Hmm... we probably need a category for each scenario (majority vs. supermajority).  I think simple majority is good for what is listed, but perhaps things like charter changes or technical direction changes need the super?

Reply:
made proposed changes that we can talk about in the TAC call today